lib/Db/View.php

```diff
@@ -122,6 +122,15 @@ public function getColumnsSettingsArray(): array {
 		return $result;
 	}
 
+	public function findColumnSettingsForColumn(int $columnId): ?ViewColumnInformation {
```

Comment:
I am wondering whether we should index the settings array by id (at least after reading it from Db) 🤔 Probably to micro to be necessary tho.

Reply:
as this PR already huge, I propose to do any refactoring about columns indexing in a separate PR :pray: